test/classes/delete-free/undecorated-generic/cannot-default-init-c.good

```diff
@@ -1,5 +1,3 @@
 cannot-default-init-c.chpl:3: In function 'test':
-cannot-default-init-c.chpl:4: error: Cannot default-initialize a variable with generic type
-cannot-default-init-c.chpl:4: note: 'bb' has generic type 'C'
-cannot-default-init-c.chpl:4: note: 'C' now means non-nilable class with any management
-cannot-default-init-c.chpl:4: note: to migrate old code, change it to 'borrowed C'
+cannot-default-init-c.chpl:4: error: 'bb' cannot be default initialized
+cannot-default-init-c.chpl:5: note: 'bb' is before being initialized here
```

Comment:
put "used" in `is used before being...` Better yet, indicate that "here" is about the use point, not the initialization point: `'bb' is used here before being initialized`